### PR TITLE
OpenCADC harbor deployment files

### PR DESCRIPTION
**Pull request by kcasteels**

### CODE REVIEW

File: deployment/k8s-config/harbor/opencadc/pvc-mount.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
```
Comment (brianmajor) [inline deployment/k8s-config/harbor/opencadc/pvc-mount.yaml]: Is this file for debugging the PVC?  Should it be in here?  If you think it should then its use should be documented.

**Review by brianmajor — CHANGES_REQUESTED** [review]
Just a few points @kcasteels :
1. Could you provide a README.md with brief install instructions?
2. Since there is now an `opencadc` folder, could you create a new folder for images.canfar.net (not what that would be) and move the existing files in there?  They should be on the same level for organization purposes.
3. And just a question about whether we need that PVC yaml file...

**Review by brianmajor — CHANGES_REQUESTED** [review]
Thanks for the reorganization, but I think you missed my comments about a README.md and there's a question about the pvc-mount.yaml file.

File: deployment/k8s-config/harbor/opencadc/pvc-mount.yaml (hunk 1 above)
Comment (kcasteels) [reply deployment/k8s-config/harbor/opencadc/pvc-mount.yaml]: I've moved this file to deployment/k8s-config/scripts directory as it is a debugging tool.

**Review by brianmajor — APPROVED** [review]

**Pull request merged**